tests/rendering/zelos/zelos.html

```diff
@@ -32,7 +32,11 @@
   <script>
     goog.require('Blockly.blockRendering.Debug');
     goog.require('Blockly.zelos.Renderer');
+    goog.require('Blockly.zelos.Themes.Classic');
     // Blockly.blockRendering.startDebugger();
+
+    Blockly.BlockSvg.prototype.setOutputShape = function() { };
```

Comment:
?

Reply:
There as a stub for pxt-blockly blocks. We might want to consider allowing blocks to choose their output shape (see https://github.com/google/blockly/issues/3414) but for now a stub will get around that for testing.

Reply:
Added comment.